code/upaya/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './public/js/entry.js',
+
+  output: {
+    filename: 'bundle.js',
+    path: __dirname + '/public/js'
+  },
+
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        query: {
+          presets: ['es2015', 'stage-0', 'react'],
+        }
+      },
+    ]
+  },
+
+  resolve: {
+    extensions: ['', '.js', '.jsx']
+  },
+
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': "'production'"
```

Comment:
ack we are :/, i'm going to remove this for now (added because react now throws a console warning/error if this env var isn't there)